core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/manytomany/ManyToManyTest.java

```diff
@@ -135,7 +135,7 @@ private void delete(Session session, Object... entities) {
 	}
 
 	private int expectedAssociationNumber() {
-		if ( TestHelper.getCurrentDialectType().equals( GridDialectType.NEO4J ) ) {
+		if ( TestHelper.getCurrentDialectType().equals( GridDialectType.NEO4J ) || TestHelper.getCurrentDialectType().equals( GridDialectType.NEO4J_REMOTE )) {
```

Comment:
Minor nitpicking but we might as well be consistent in the way we test for this. See the change just above this one.